packages/root-cms/ui/pages/DocumentPage/DocumentPage.tsx

```diff
@@ -36,10 +34,14 @@ interface DocumentPageProps {
 function getPreviewUrl(
   collectionId: string,
   slug: string,
-  selectedLocale = ''
+  selectedLocale = '',
+  embed = true
 ) {
   const basePreviewPath = getDocPreviewPath({collectionId, slug});
   const searchParams = new URLSearchParams(window.location.search);
+  if (embed) {
+    searchParams.set('embed', 'true');
```

Comment:
isn't there a way to detect if the page is within an iframe without this query param? and maybe you can check referrer url starts with `/cms/`

Reply:
looks like this worked! had to modify the "open in new tab" link to remove the referer